torch/_inductor/ir.py

```diff
@@ -4009,7 +4011,7 @@ def convert_to_reinterpret_view(cls, x):
 
     @classmethod
     def realize_input(cls, x):
-        if x is None:
+        if x is None or isinstance(x, EffectfulKernel):
```

Comment:
Hmm, I don't think we should need this change. We shouldn't ever be realizing a token, or the output of print. Maybe if you remove SinkTokens you'll no longer need this change?

If we are realizing a TorchBind class then the realizing should work as with other ExternKernels

Reply:
Why do we need to do this ? I don't think we should